service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java

```diff
@@ -168,4 +179,57 @@ public void inviteToTeam(JWTData userData, InviteMembersRequest inviteMembersReq
       emailer.sendInviteEmail(email, "Team Member", inviter, inviterTeam);
     });
   }
+
```

Comment:
I like how organized these methods are but we're sacrificing a lot of speed by making so many separate database calls. Let's get it merged in as is but I suspect we'll have to come back to combine some of these calls later to speed things up